pages/common/git-bisect.md

```diff
@@ -0,0 +1,20 @@
+# git bisect
+
+> Use binary search to find the commit that introduced a bug.
+> Git automatically jumps back and forth in the commit graph to progressively narrow down the faulty commit.
+
+- Start a bisect section by passing a known good commit and a known bad commit:
```

Comment:
I think that makes sense. Will change.

ps - I just realized I meant to write "Start a bisect session", not "section"! Will fix that as well.